CONTRIBUTING.md

```diff
@@ -66,38 +77,31 @@ When you make changes to your pull request, please *do not close and reopen your
 #### Travis
 
 Once you have started a pull request, you will get instant feedback from Travis. It will check if the templates you have changed
-comply with the our guidelines. At the moment not all packages comply to the rules, so if you update a package, it may happen, that Travis
-reports errors on places you haven't touched. So feel free to fix those errors too.
-
-You are encouraged to check your templates beforehand using the helper from the [xtools](https://github.com/chneukirchen/xtools) package:
-
-    $ xlint template
+comply with the our guidelines. At the moment not all packages comply with the rules, so if you update a package, it may happen that Travis reports errors about places you haven't touched. Please feel free to fix those errors too.
 
 #### Review
 
-GitHub reports new pull request at our IRC-Channel, so the reviewers will be instantly informed. Most of the time
-your pull request will contain mistakes. It's nothing bad, it just happens.
+Most of the time your pull request will contain mistakes. It's nothing bad, it just happens.
 
-The reviewers will comment your pull request and point out which changes are needed before the template can be included.
+Reviewers will comment your on pull request and point out which changes are needed before the template can be included.
 
 We recommend having only a single commit for pull request, so if you need to make changes in commits but already have a pull request, use the following commands:
 
-
     $ git add <file>
     $ git commit --amend
     $ git push -f
 
 #### Closing the pull request
 
-Once you have applied all comments, the reviewers will merge your request.
+Once you have applied all changes requested in reviewer comments, the reviewers will merge your request.
 
-If the pull request gets inactive for some days, the reviewers may or may not warn you when they are about to close it.
-If it stays inactive further, it'll be closed.
+If the pull request becomes inactive for some days, the reviewers may or may not warn you when they are about to close it.
+If it stays inactive further, it will be closed.
 
-Please abstain from temporary closing a pull request while revising the templates. Only close your pull request if
+Please abstain from temporarily closing a pull request while revising the templates. Only close your pull request if
```

Comment:
While touching this, please add what to do instead of closing - commenting what's wrong or adding "[WIP]" to PR title.

Reply:
Thanks - i'll change that to:

> the templates. Instead, leave a comment on the PR describing what still needs work, or add "[WIP]" to the PR title. Only ...